services/chrome-web-store/chrome-web-store-rating.service.js

```diff
@@ -1,33 +1,131 @@
 'use strict'
 
-const { deprecatedService } = require('..')
+const { floorCount: floorCountColor } = require('../color-formatters')
+const { metric, starRating } = require('../text-formatters')
+const { NotFound } = require('..')
+const BaseChromeWebStoreService = require('./chrome-web-store-base')
 
-const commonAttrs = {
-  category: 'rating',
-  label: 'rating',
-  dateAdded: new Date('2020-09-06'),
+class BaseChromeWebStoreRating extends BaseChromeWebStoreService {
+  static get category() {
```

Comment:
Could you please convert this to static fields? See #5555 for more information.

Reply:
@PyvesB Done.